loco-new/base_template/src/tasks/user_delete.rs

```diff
@@ -0,0 +1,69 @@
+use crate::models::_entities::users;
+use loco_rs::prelude::*;
+use std::env;
+
+pub struct UserDelete;
+#[async_trait]
+impl Task for UserDelete {
+    fn task(&self) -> TaskInfo {
+        TaskInfo {
+            name: "user:delete".to_string(),
+            detail: "Delete a user by entering email.\nUsage:\ncargo run task user:delete"
+                .to_string(),
+        }
+    }
+    async fn run(&self, app_context: &AppContext, vars: &task::Vars) -> Result<()> {
+        let input = match vars.cli_arg("pid") {
+            Ok(email) => email,
+            Err(_) => return Err(Error::string("pid is mandatory")),
+        };
+
+        let user_to_delete = users::Model::find_by_pid(&app_context.db, &input).await?;
+
+        tracing::info!(
+            message = "User to delete",
+            user_email = &user_to_delete.email,
+            user_pid = user_to_delete.pid.to_string(),
+            "delete user via task"
+        );
+
+        let mut confirm = String::new();
+
+        // If test ist running simulate confirmation
+        if let Ok(var) = env::var("TEST_CAN_RUN_USER_DELET_BY_PID") {
```

Comment:
don’t think we should rely on a test-only environment variable (TEST_CAN_RUN_USER_DELET_BY_PID) in production code. It couples test behavior into runtime logic, is easy to forget to unset, and can be risky for a destructive task.

A more idiomatic and safer approach would be to add an explicit CLI flag such as force:true to bypass the confirmation prompt. This is a common convention for destructive commands, keeps intent explicit, and allows tests to run without special env vars.